### ISSUE TITLE
ui5-busy-indicator crashes in ui5-responsive-popover during initial render

### ISSUE THREAD

**Issue by charlie-maloney** [SAP/ui5-webcomponents#4412]
### **Bug Description**
When rendering an ui5-responsive-popover for the first time and immediately opening it with an active ui5-busy-indicator inside, the site crashes with a JS runtime error:

`TypeError: Cannot read properties of null (reading 'focus')`

Think we should check to make sure we have element before we call `focus()`. i.e.

```js
this.shadowRoot.querySelector(".ui5-busy-indicator-busy-area")?.focus();
```

### **Expected Behavior**

Busy Indicator should be active and no error should be thrown
(almost exact same issue as https://github.com/SAP/ui5-webcomponents/issues/4108)
@MarcusNotheis FYI

### **Steps to Reproduce**
- Open https://codesandbox.io/s/table-dialog-busy-forked-6l1zd?file=/src/App.js
- Click "Show Dialog" button
- see the error

### **Isolated Example**
https://codesandbox.io/s/table-dialog-busy-forked-6l1zd?file=/src/App.js

### **Context**
- UI5 Web Components version: 1.0.0-rc.15 (and latest snapshots as well)
- OS/Platform: macOS
- Browser: Chrome
- Affected component: ui5-busy-indicator

### **Log Output / Stack Trace / Screenshots**
```
TypeError: Cannot read properties of null (reading 'focus')
HTMLElement._redirectFocus
node_modules/@ui5/webcomponents/dist/BusyIndicator.js:265
  262 | 		}
  263 | 
  264 | 		event.preventDefault();
> 265 | 		this.shadowRoot.querySelector(".ui5-busy-indicator-busy-area").focus();
  266 | 	}
  267 | }
  268 | 
```
### **Priority**
- [ ] Low
- [ ] Medium
- [x] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

**Comment by gmkv** [comment]
Hi @charlie-maloney 

I retested your example with the a build using the last commit in the main branch and I didn't get a crash. The PR which closes #4108 also solves your error too.
You can consume the fix in the next patch release of the ui5-webcomponents library!

Regards,
Georgi

**Comment by charlie-maloney** [comment]
> Hi @charlie-maloney
> 
> I retested your example with a build using the last commit in the main branch and I didn't get a crash. The PR which closes #4108 also solves your error too. You can consume the fix in the next patch release of the ui5-webcomponents library!
> 
> Regards, Georgi

Great, thanks @gmkv!
